packages/salesforcedx-vscode-lwc/src/commands/lightningLwcPreview.ts

```diff
@@ -144,24 +143,31 @@ export const lwcPreview = async (sourceUri: URI) => {
     return;
   }
 
-  if (!fs.existsSync(resourcePath)) {
-    const message = nls.localize('lightning_lwc_preview_file_nonexist', resourcePath);
-    showError(new Error(message), logName, commandName);
-    return;
-  }
+  try {
+    const stat = await vscode.workspace.fs.stat(sourceUri);
+    if (!stat) {
```

Comment:
is this possible with fs.stat?  Under what circumstances would it return something falsy?

Reply:
good catch! fs.stat would just throw an error